commands/audit/auditparams.go

```diff
@@ -232,6 +232,8 @@ func (params *AuditParams) ToBuildInfoBomGenParams() (bomParams technologies.Bui
 		PipRequirementsFile: params.PipRequirementsFile(),
 		// Pnpm params
 		MaxTreeDepth: params.MaxTreeDepth(),
+		// Docker params
+		DockerImageName: params.DockerImageName(),
```

Comment:
should not be added to audit....
curation command does not use this funcm it builds its own params with values at `ca.getBuildInfoParamsByTech()`
it should be added only there